mobile/build.gradle

```diff
@@ -12,12 +12,12 @@ ext {
 }
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 29
     defaultConfig {
         applicationId "org.hzontal.tella"
         minSdkVersion 19
         compileSdkVersion 29
-        targetSdkVersion 30
+        targetSdkVersion 29
```

Comment:
No need to downgrade

Reply:
It was the only way to show the other modules in my Android Studio. I answered @ahlem-jarrar about this.
Or is there another way?
You can abandon that commit.

Reply:
Those two things shouldn't be connected. Not sure, but it may be necessary to run the Gradle build to see all the modules.

Just remove build.gradle and google-services.json files from the PR.